Nos/Views/Discover/FeaturedAuthor.swift

```diff
@@ -229,3 +205,343 @@ extension FeaturedAuthor {
+            npub: "npub12rze589jx0gg6kslkjfl2gxxkhtlw73t5shyve5qrglrv6c2qflqejj7ns",
+            categories: [.photography]
+        ),
+        FeaturedAuthor(
+            name: "Ratel",
+            npub: "npub15c3sa5kzedkfys95mc5egmst8ptuqefvezemae0883un7hqf5d5sgmqlxf",
+            categories: [.art]
+        ),
+        FeaturedAuthor(
+            name: "Wellington Chinese History",
+            npub: "npub1fdyax3st9gcqj7zhqres3fw9pf7qrkukppv3fs7w2mcjcsqyrxjqcgcxcn",
+            categories: [.newzealand]
+        ),
+        FeaturedAuthor(
+            name: "David Hood",
+            npub: "npub12aew64q5lsxpcc98lskha0564gtjn30cw7vdueqnyj06xjqxtmksahdrlg",
+            categories: [.newzealand, .news]
+        ),
+        FeaturedAuthor(
+            name: "Kris Sowersby",
+            npub: "npub1t6n8u5sa80m6sx53m7uge6afktek4me982wvclr3xe5tfeyg34fslkxqcz",
+            categories: [.art, .newzealand]
+        ),
+        FeaturedAuthor(
+            name: "Greenpeace Atoearoa",
+            npub: "npub1mn78z9rssfq2yjqdwsa3h4qwu6zpw4xku9ppfldkde4ycwnpch4qhmcssl",
+            categories: [.activists, .newzealand]
+        ),
+        FeaturedAuthor(
+            name: "Bill Bennett",
+            npub: "npub1gpcxn339ap3r9tju2uy6d5k8rmud7f8kr579w7zv54qyevgz9xsqdtr4us",
+            categories: [.news, .newzealand]
+        ),
+        FeaturedAuthor(
+            name: "The Spinoff",
+            npub: "npub1qngl40pxy06k3hrqp4ltkn4p5yacen7u93du58v4tamf73tzaqhs6dfy8g",
+            categories: [.news, .newzealand]
+        ),
+        FeaturedAuthor(
+            name: "Dan Selvin",
+            npub: "npub16jjukmhnvflj92dv2jr8z6udfhzyyuyf3mckmf6afws9w4xdhhzs745kmv",
+            categories: [.news, .newzealand]
+        ),
+        FeaturedAuthor(
+            name: "Emma Cook",
+            npub: "npub18pl5kgxl47e5pssxl9q38m2v7ez477gw3nz220jz2vslnjreqq8s2yxl9m",
+            categories: [.art, .newzealand]
+        ),
+        FeaturedAuthor(
+            name: "Jack Yan",
+            npub: "npub179lhhjz30t3n4utkkn26qzgmpx44v9s5qdtwjs5686px8ulw7dfszxmfgp",
+            categories: [.news, .newzealand]
+        ),
+        FeaturedAuthor(
+            name: "Lucire",
+            npub: "npub1l4s4mtt96znwu3plfeyupk37y63xfapw5e7kjn7uavuw02lvav5q3ngc3s",
+            categories: [.news, .newzealand]
+        ),
+        FeaturedAuthor(
+            name: "Ian Griffin",
+            npub: "npub19p6edn6lvz9dmegm7s8vyz9w5w0ejzgztke8rcswxgmqr35h8feszt36jc",
+            categories: [.scifi, .newzealand]
+        ),
+        FeaturedAuthor(
+            name: "JSM",
+            npub: "npub1uqeexjx2djkfwzxdnrnrrch5h2k4xn0uapcgsxm94ftaxrlhy5lqywjckg",
+            categories: [.tech]
+        ),
+        FeaturedAuthor(
+            name: "Carla",
+            npub: "npub1hu3hdctm5nkzd8gslnyedfr5ddz3z547jqcl5j88g4fame2jd08qh6h8nh",
+            categories: [.random]
+        ),
+        FeaturedAuthor(
+            name: "WaveLake",
```

Comment:
```suggestion
            name: "Wavlake",
```

Reply:
🧙‍♂️

Reply:
@pelumy Would you mind checking the others here to ensure their names are correct? It's possible some of the names in the spreadsheet were wrong. If you search by npub for each author and check the name, that would be really nice. I'd love for this list to be accurate, even though the names are just there for the developers and don't appear in the app at all.